### ISSUE TITLE
Precompilation issue with Julia v1.6

### ISSUE THREAD

**Issue by Nosferican** [sambitdash/PDFIO.jl#95]
When precompiling PDFIO with Julia v1.6 it has just keep hanging for hours. Tested it with Julia `v1.6.0-beta1` and PDFIO `v0.1.10`.

**Comment by sambitdash** [comment]
@Nosferican we will review the PDFIO compatibility issues with any release of Julia after the version is generally available.

**Comment by Nosferican** [comment]
I suspect the issue might be due to some change in the pre-compilation / artifacts system so it would be good to address it while in beta so it can be fixed for the release candidate (in case the issue is in Base or a stdlib and not in this package). The v1.6 beta has binary files to install from the website https://julialang.org/downloads/#upcoming_release. The beta version is also available to use in CI (see [Discourse](https://discourse.julialang.org/t/julia-v1-6-0-beta1-is-now-available/53065)). It seems the issue is currently not known to the core maintainers as it was not included in the PkgEval summary (see [Discourse](https://discourse.julialang.org/t/analysis-of-pkgeval-results-for-upcoming-1-6/53194)).

**Comment by sambitdash** [comment]
It's already captured and can be seen in the logs:

https://s3.amazonaws.com/julialang-reports/nanosoldier/pkgeval/by_hash/56ff37a_vs_ebcc603/PDFIO.1.6.0-beta1-73f862b30f.log

The issue known and already has been addressed in the trunk and now tagged as v0.1.11 and should work normally. 

Since, there was a failure across version due to an access issue to test link, I fixed it and released version v0.1.11.

Any further issues with the Julia@1.6 will be looked at after the release if the version.
